libs/domains/application/src/lib/slices/applications.slice.ts

```diff
@@ -41,24 +44,37 @@ const applicationDeploymentsApi = new ApplicationDeploymentHistoryApi()
 const applicationMetricsApi = new ApplicationMetricsApi()
 const applicationConfigurationApi = new ApplicationConfigurationApi()
 
-export const fetchApplications = createAsyncThunk<Application[], { environmentId: string; withoutStatus?: boolean }>(
-  'applications/fetch',
-  async (data, thunkApi) => {
-    const response = await applicationsApi.listApplication(data.environmentId)
-
-    if (!data.withoutStatus) {
-      thunkApi.dispatch(fetchApplicationsStatus({ environmentId: data.environmentId }))
-    }
-
-    return response.data.results as Application[]
+const containersApi = new ContainersApi()
+
+export const fetchApplications = createAsyncThunk<
+  Application[] | ContainerResponse[],
+  { environmentId: string; withoutStatus?: boolean }
+>('applications/fetch', async (data, thunkApi) => {
+  // fetch Git applications
+  const responseApplication = (await applicationsApi.listApplication(data.environmentId)).data
```

Comment:
I can't trigger without await, the only solution is to add two requests reduxThunk requests with two different LoadingStatus otherwise it should be a glitch.

I left it like that for the moment but it will be quick to change if we ever discuss it again